participants/minh_lengoc.json

```diff
@@ -0,0 +1,42 @@
+// Please provide your info in your own .json file.
+// See https://jscraftcamp.org/registration for more information
+{
+	// your name / nickname (required)
+	"name": "Minh Le Ngoc",
+	// optional company name (required)
+	"company": "Ambright",
+	// either both days or at least one day has to be set to true
+	"when": {
+		// 2023-06-30
+		"friday": true,
+		// 2023-07-01
+		"saturday": false
+	},
+	// if you are willing to take session notes and publish them to github (required)
+	"iCanTakeNotesDuringSessions": false,
+	// your current interests (JS and in general) (required)
+	"tags": ["ReScript", "Svelte", "Zustand", "GraphQL"],
+	// if you only eat vegan food (optional)
+	"vegan": false,
+	// if you only eat vegan or vegetarian food (optional)
+	"vegetarian": false,
+	// what you cannot eat or drink (optional)
+	"allergies": ["Crab"],
+	// tell us a few words how JavaScript affects you (required)
+	"whatIsMyConnectionToJavascript": "Workingstudent",
+	// what can you contribute to the bar camp (required)
+	"whatCanIContribute": "Nothing, just listen and learn from experts?",
```

Comment:
Everybody can contribute something 🙂

For example: "I bring the perspective of a junior JS dev and I will ask questions that make you think about simpler solutions"

If you want to listen and learn, why not try taking and sharing notes? 🙂